pom.xml

```diff
@@ -39,6 +39,11 @@
             <groupId>io.jenkins.plugins</groupId>
             <artifactId>ionicons-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.kohsuke</groupId>
+            <artifactId>access-modifier-suppressions</artifactId>
+            <version>1.33</version>
+        </dependency>
```

Comment:
I think the typical pattern (pending https://github.com/jenkinsci/plugin-pom/issues/904) to avoid embedding the JAR is:

```suggestion
        <dependency>
            <groupId>org.kohsuke</groupId>
            <artifactId>access-modifier-suppressions</artifactId>
            <version>${access-modifier-checker.version}</version>
            <!-- Not needed at runtime -->
            <scope>provided</scope>
            <exclusions>
                <exclusion>
                    <!-- Upper bounds and comes from core -->
                    <groupId>org.kohsuke</groupId>
                    <artifactId>access-modifier-annotation</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```